posthog/models/team/team.py

```diff
@@ -168,6 +169,10 @@ def get_default_data_attributes() -> list[str]:
     return ["data-attr"]
 
 
+def get_default_modifiers() -> Mapping[str, Any]:
```

Comment:
I'm only worried about us  creating two ways of defining default modifiers here:
1. this, at team creation time,
2. and the existing `set_default_modifier_values`, at query time.
What do you think about setting `propertyGroupsMode` in `set_default_modifier_values()` based on `team.created_at`? One bonus would be enabling `propertyGroupsMode` retroactively, for team created a few days back let's say